pkgs/top-level/cubocore-packages.nix

```diff
@@ -0,0 +1,57 @@
+{ makeScope, newScope, lxqt }:
```

Comment:
This file is not called anywhere.

Reply:
I'm just the first time to have a deal with scoop. Sorry if that for mistakes.